content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -304,13 +304,23 @@ specify a readiness probe. In this case, the readiness probe might be the same
 as the liveness probe, but the existence of the readiness probe in the spec means
 that the Pod will start without receiving any traffic and only start receiving
 traffic after the probe starts succeeding.
-If your container needs to work on loading large data, configuration files, or
-migrations during startup, specify a readiness probe.
 
 If you want your container to be able to take itself down for maintenance, you
 can specify a readiness probe that checks an endpoint specific to readiness that
 is different from the liveness probe.
 
+If your app has a strict dependency on back-end services, you can implement both
+a liveness and a readiness probe. The liveness probe passes when the app itself
+is healthy, but the readiness problem additionally checks that each required
+back-end service is available. This helps you avoid directing traffic to Pods
+that can only respond with error messages.
+
+If your container needs to work on loading large data, configuration files, or
```

Comment:
nit: ... loading large amounts of data OR
loading large data or configuration files

Reply:
(I'd like to keep the LGTM). I'm not sure what the whole revised sentence would look like @kbhawkey - if you could write that out, that'll help me if I do a follow-up PR to tweak it.

Reply:
👍